### ISSUE TITLE
Homography pretraining not converging

### ISSUE THREAD

**Issue by ikaftan** [cvg/glue-factory#53]
Hi! First of all, thank you for making the training and evalution code open-source. I am currently training LightGlue with the `superpoint+lightglue_homography.yaml` config using a batch size of 128 on a single Nvidia Tesla A100 GPU with 40 GB VRAM. I use `python/3.8.5`, `cuda/11.8.0`, `cudnn/8.4.0.27`, and `torch/2.1.2`.

However, I have observed that the total validation/training loss does not decrease below 0.8 during homography pretraining. This result is different compared to Figure 5 in the paper where the loss goes below 0.2. I am seeking insights or suggestions on why this discrepancy might be occurring. Are there any specific considerations or adjustments to the configuration that I might be overlooking?

**Comment by zjs2121230251** [comment]
May I ask whether the “loss” referred to in Fig. 5 of the paper corresponds to loss/last
